lib/cisco_node_utils/tacacs_server_host.rb

```diff
@@ -23,41 +23,107 @@ class TacacsServerHost < NodeUtil
     attr_reader :name
     @hosts = {}
 
-    def initialize(name, create=true)
+    def initialize(name, instantiate=true, host_port=nil)
```

Comment:
@chrisvanheuveln Please refer to @glennmatthews for further details. This is the same issue that we had with radius_server (which is merged). Basically, the port forms part of the "uniqueness" of the host, and thus must be provided at creation time.